test/unit/state/siteUtilTest.js

```diff
@@ -286,6 +286,14 @@ describe('siteUtil', function () {
           assert.equal(!!processedSites.getIn([0, 'lastAccessedTime']), true)
           assert.deepEqual(processedSites.getIn([0, 'tags']).toJS(), [])
         })
+        it('returns true for bookmarked entries', function () {
+          const siteDetail = Immutable.fromJS({
+            location: testUrl1,
+            lastAccessedTime: 123,
+            tags: [siteTags.BOOKMARK]
+          })
```

Comment:
I guess you might forgot to call `addSite` for this siteDetail and you may need to leverage previous `processedSites` as first argument.